crosslink/internal/common.go

```diff
@@ -15,27 +15,28 @@
 package crosslink
 
 import (
+	"fmt"
 	"os"
 	"path/filepath"
 
 	tools "go.opentelemetry.io/build-tools"
 	"golang.org/x/mod/modfile"
 )
 
-func writeModules(module moduleInfo) error {
+func writeModule(module moduleInfo) error {
 	mfParsed, err := modfile.Parse("go.mod", module.moduleContents, nil)
 	if err != nil {
-		return err
+		return fmt.Errorf("additional info: %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("unable to parse go.mod: %w", err)
```

I think you may have taken my comment about adding additional info to errors a bit too literally! :)

Reply:
oops